tests/testthat/test-json_nest.R

```diff
@@ -5,3 +5,21 @@ test_that("`json_nest()` works", {
     nested
   })
 })
+
+test_that("`json_nest()` works remotely", {
+  # FIXME: add "mssql" when ready
+  skip_if_src_not("postgres")
+  con <- my_test_src()$con
+
+  withr::defer(
+    try(dbExecute(con, "DROP TABLE iris"))
+  )
+
+  dbWriteTable(con, "iris", iris)
+  iris_remote <- tbl(con, "iris")
```

Comment:
We have `test_db_src_frame()` for that. I wouldn't mind using a simpler test data frame here, and also avoid iris for new code.

Reply:
https://github.com/cynkra/dm/pull/975/commits/7c030ebc5cb1e910ddc3f2ca234713888fa0cab4